iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -467,6 +503,8 @@ TEST_F(PortManager_test, AcquiringOneMoreThanMaximumNumberOfNodesFails)
         iox::NodeName_t newNodeName(iox::cxx::TruncateToCapacity, node + std::to_string(i));
         auto nodeData = m_portManager->acquireNodeData(newProcessName, newNodeName);
         ASSERT_THAT(nodeData.has_error(), Eq(false));
+        EXPECT_THAT(nodeData.value()->m_node, StrEq(newNodeName));
+        EXPECT_THAT(nodeData.value()->m_process, StrEq(newProcessName));
```

Comment:
Please don't merge these two tests. One is for acquiring the maximum number of nodes and one for the correct overflow handling.
Please have a look at #547 for the reason.